### ISSUE TITLE
Can't parse declaration starting with 'as' after import

### ISSUE THREAD

**Issue by ollef** [stil4m/elm-syntax#29]
I was using `elm-analyse` (which is great by the way, thanks for that!), and ran into a parse failure. I think this is the appropriate place to file the issue.

Here's a small module that doesn't parse with `elm-analyse`:

```elm
module Test exposing (asSomething)

import Import


asSomething : Int
asSomething =
    123
```

If `asSomething` is renamed to something that doesn't start with `as`, it works, or if I do `import Import as Import`, so it seems that the import parser is greedily munching away on the `as` of the declaration when it expects a possible `as`.

Cheers!

**Comment by Viir** [comment]
There is another constraint on how the `as` can appear. While playing with different syntax in [ellie](https://ellie-app.com/4vv9KwgLMCya1), I found that the `as` needs to be indented, with at least one whitespace.

The following code compiled:
```elm
import Browser
 as OtherName
```
While the following code resulted in a Parse Error:
```elm
import Browser
as OtherName
```
An example source code can be found at https://ellie-app.com/4vv9KwgLMCya1

**Comment by stil4m** [comment]
Thanks for the report and good examples.
This is resolved in version `7.0.6`
